src/api-calls.js

```diff
@@ -0,0 +1,9 @@
+export const getMovie = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+}
+
+export const getVideo = ( id ) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`)
+    .then(response => response.json())
+}
```

Comment:
What about if the fetch fails? might be good to add some sad path operations here. Additionally - these fetches look nearly identical - can you think of any way to DRY them up into one, using args and params?